### ISSUE TITLE
Not compatible with python3 (Forked)

### ISSUE THREAD

**Issue by Zeioth** [aaronbassett/DisposableEmailChecker#5]
I recieve this error using it with django 1.7.1 under python3 when i invoke is_disposable() method.
"name 'xrange' is not defined"

**Comment by Zeioth** [comment]
Forked as django-email-blacklist

**Comment by aaronbassett** [comment]
Hey Zeioth, sorry it took me a while to get back to you. There is a new version of the package under the rewrite branch (#e8667efb1837b4f1821eb6bb6d0039602be28269) which fixes the Python 3 error (it now supports 2.7, 3.3 and 3.4 with Tox for testing) as well as a lot of other problems. However it's not backwards compatible as I've converted it into a Django model field and validator so it's actually easier to use.

```python
class MyModel(models.Model):
    email = DisposableEmailField()
```

If you want to continue with your fork I completely understand, but once I'll be merging this branch into master and pushing to pypi later today.
